server/channels/app/user.go

```diff
@@ -53,7 +53,9 @@ func (a *App) CreateUserWithToken(c request.CTX, user *model.User, token *model.
 	}
 
 	if model.GetMillis()-token.CreateAt >= InvitationExpiryTime {
-		a.DeleteToken(token)
+		if appErr := a.DeleteToken(token); appErr != nil {
+			c.Logger().Warn("Error while deleting token", mlog.Err(appErr))
```

Comment:
Can we make the log message a bit more verbose? What was the token getting deleted?